backend/src/nodes/utils/image_utils.py

```diff
@@ -138,6 +144,39 @@ def normalize_normals(
     return x, y, z
 
 
+def select_fill(img: np.ndarray, fill: int, for_pil: bool = False):
```

Comment:
Instead of having a strange `for_pil` parameter, couldn't `rotate` just multiply the numbers it gets by 255?

If we ever add more fill colors, the `for_pil` will make things more messy.

Reply:
Also, why `select_fill`? Wouldn't `get_fill_color` be more descriptive? "Select" sounds like this function has side effects to me.

Reply:
Because it selects a fill color based on the option. 'Get' works too.